### ISSUE TITLE
Enhance the service list view with the host icon images

### ISSUE THREAD

**Issue by EmmaTinten** [Icinga/icingaweb2#4475]
## Describe the solution you'd like
In the service list we want to display the host icons direclty in front of the host name.

It would allow an easy and quick view on the service list to decide if the sql service on host xyz is to be reported to the windows guys because it is running on a host with the win icon configured. So, the additional display of the host icon image would be great.

**Comment by nilmerg** [comment]
That's the case. For hosts and services.

**Comment by EmmaTinten** [comment]
Not really. Using Icinga Web 2 Version 2.8.2 and viewing /icingaweb2/monitoring/list/services the list only contains configured service icons. I want to see the host icon image too.

**Comment by nilmerg** [comment]
Then check your configuration please. It's working fine for me:

![Screenshot from 2021-07-21 14-16-59](https://user-images.githubusercontent.com/16668527/126487097-8e1e9534-3ff4-4534-9c92-7d80885d006c.png)

**Comment by EmmaTinten** [comment]
Please check the url, I am visiting monitoring/list/services not the host details.

See host4 with configured icon:

![image](https://user-images.githubusercontent.com/45063741/126487356-8c9a2ccb-a5cb-416f-b8fc-98fede5f7a14.png)

This is the image of list/services where I want to see the host icon too:

![image](https://user-images.githubusercontent.com/45063741/126487477-642932e8-bf74-4733-b7c9-1ba987ec14f6.png)

**Comment by nilmerg** [comment]
I see. :facepalm:

**Comment by flourish86** [comment]
Thanks for your input! While the additional icon in the title might help some users, we found that other users prefer the clearer version.

But with Icinga DB Web we already implemented a view mode switcher for lists, that let users choose the level of detail information of object lists. We're going to follow that direction further and plan to offer more options to customize list views. This could also contain a toggle to show/hide the icon images.

Here's a quick draft on how this might look. The final version might differ, though.

![Customize Controls](https://user-images.githubusercontent.com/6977434/127124831-32c694ea-b114-4e6e-813b-fc4adbb936f8.jpg)
